application/src/main/java/org/togetherjava/tjbot/commands/tags/TagsCommand.java

```diff
@@ -26,6 +30,8 @@
  * </pre>
  */
 public final class TagsCommand extends SlashCommandAdapter {
+    private static final Logger logger = LoggerFactory.getLogger(TagsCommand.class);
+    private static final int MAX_TAGS_THRESHOLD_WARNING = 200;
     private final TagSystem tagSystem;
```

Comment:
Put a space here to separate the static and non-static fields.

Reply:
Done